src/modules/audio/audio.c

```diff
@@ -198,6 +337,13 @@ void lovrAudioDestroy() {
   }
   arr_free(&state.converters);
   memset(&state, 0, sizeof(state));
+  free(state.scratchBuffer1); state.scratchBuffer1 = NULL;
+  free(state.scratchBuffer2); state.scratchBuffer2 = NULL;
+  free(state.persistBuffer);  state.persistBuffer = NULL;
+
+#ifdef LOVR_DEBUG_AUDIOTAP
+  state.audiotapWriting = false;
+#endif
```

Comment:
The buffers aren't getting freed here.  The memset clears `state`, so free is getting NULL.  Can move the frees above the memset and skip the explicit setting to NULL/false.